Common/src/main/java/com/blamejared/crafttweaker/api/tag/CraftTweakerTagRegistry.java

```diff
@@ -296,4 +319,25 @@ public String makeTagFolder(ResourceKey<?> key) {
         return tagDir;
     }
     
+    public static class BindContext {
+        
+        private boolean registerKnownManagers = true;
+        
+        public BindContext() {
+        
+        }
+    
+        public boolean registerKnownManagers() {
+        
+            return registerKnownManagers;
+        }
+    
+        public BindContext registerKnownManagers(boolean value) {
+            
+            this.registerKnownManagers = value;
+            return this;
+        }
+        
+    }
+    
 }
```

Comment:
Any reason why this is not a record and is instead mutable?

Reply:
Future proofing in case we need to add more to the context

Reply:
You can still use a record and overload the constructor if needed